common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts

```diff
@@ -212,6 +212,45 @@ export class StorageAccounts {
         return storageAccounts[index];
     }
 
+    public async getStorageAccountProperties(resourceGroupName: string, storageAccountName: string): Promise<Model.StorageAccount | undefined> {
+        if (resourceGroupName === null || resourceGroupName === undefined || typeof resourceGroupName.valueOf() !== 'string') {
+            throw new Error(tl.loc("ResourceGroupCannotBeNull"));
+        }
+
+        if (storageAccountName === null || storageAccountName === undefined || typeof storageAccountName.valueOf() !== 'string') {
+            throw new Error(tl.loc("StorageAccountCannotBeNull"));
+        }
+
+        let response = await this.sendRequest(
+            "GET",
+            "//subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{storageAccountName}",
+            {
+                '{resourceGroupName}': resourceGroupName,
+                '{storageAccountName}': storageAccountName
+            }
+        );
+        
+        if (response.statusCode == 200) {
+            return response.body;
+        }
+
+        response = await this.sendRequest(
```

Comment:
Maybe a small comment might be helpful for future developers as why we're calling this endpoint.
e.g. `// call for Microsoft.ClassicStorage`

Reply:
Added comment and removed {subscriptionId} argument, thanks!